src/method.rs

```diff
@@ -335,6 +347,29 @@ mod extension {
             Ok(InlineExtension(data, src.len() as u8))
         }
 
+        /// Convert static bytes into an `InlineExtension`.
+        ///
+        /// Only the first 15 characters are used.
+        ///
+        /// # Panics
+        ///
+        /// If the input bytes are not a valid method name.
+        pub const fn from_static(src: &'static [u8]) -> InlineExtension {
+            let mut i = 0;
+            let mut dst = [0u8;15];
```

Comment:
Looks like the loop doesn't do anything if the `src` is longer than 15, so would it just truncate? Probably worth having the unit tests include a much longer string passed to `from_static`.